docs/docs/deploy/omnibus/README.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Omnibus
+sidebar_position: 3
+---
+
+:::caution
+Due to Omnibus being EOL'd by Chef in 2024, Docker is now the preferred method
+of deploying Firezone. See the [Docker deployment guide](../docker) to get
+started.
+
+Read below to continue with an Omnibus-based deployment.
+:::
+
+Firezone can be deployed on a server running a supported [Linux distribution
+](supported-platforms) in a few minutes with our Debian or Red Hat package.
+This guide will walk you through the steps to get started.
+
+## Step 1: Prerequisites
+
+* Ensure you're on a [supported Linux distribution](supported-platforms).
+  A kernel upgrade may be required to ensure WireGuard® is available.
+* Ensure port forwarding is enabled on your firewall.
+  The default Firezone configuration requires the following ports to be open:
+  * `80/tcp` (optional): For automatically issuing SSL certificates.
+  * `443/tcp`: To access the web UI.
+  * `51820/udp`: VPN traffic listen port.
+
+:::caution
+Before deploying Firezone in **production**, you'll need a valid DNS record
+pointing to this instance. See [Prepare to Deploy](../#prepare-to-deploy)
+if you haven't done this already.
+:::
+
+:::note
+Firezone modifies the kernel netfilter and routing tables.
+Other programs that modify the Linux routing table or firewall may interfere
+with Firezone’s operation. For help troubleshooting connectivity issues, see the
+[troubleshooting guide](../../administer/troubleshoot).
+:::
+
+## Step 2: Install Server
+
+Firezone Server is the portal and VPN endpoint your users
```

Comment:
I think this is what the sentence is supposed to be: `Firezone Server is the portal and VPN endpoint your users` -> `Firezone Server is the portal and VPN endpoint your your users.`